sql/core/src/test/scala/org/apache/spark/sql/errors/QueryExecutionErrorsSuite.scala

```diff
@@ -133,8 +133,8 @@ class QueryExecutionErrorsSuite extends QueryTest with SharedSparkSession {
         .agg(sum($"sales.earnings"))
         .collect()
     }
-    assert(e2.getMessage === "The feature is not supported: " +
-      "literal for '[dotnet,Dummies]' of class " +
-      "org.apache.spark.sql.catalyst.expressions.GenericRowWithSchema.")
+    assert(e2.getMessage ===
+      "The feature is not supported: pivoting by values of the column " +
+      """'[dotnet,Dummies]' of the type 'struct(lower(sales.course), training)'.""")
```

Comment:
Is this `struct(lower(sales.course), training)` a type? or a column name?

Reply:
It was actually the column name and I fixed it.
With the latest change the datatype of the column is thrown.